windows/internal/publish.bat

```diff
@@ -0,0 +1,49 @@
+@echo off
+
+set SRC_DIR=%~dp0
+pushd %SRC_DIR%
+
+set PUBLISH_BRANCH=%PACKAGE%_%DESIRED_PYTHON%
+
+git clone %ARTIFACT_REPO_URL% > nul 2>&1
+
+IF ERRORLEVEL 1 (
+    echo Clone failed
+    exit /b 1
+)
+
+cd pytorch_builder
+attrib -s -h -r . /s /d
+
+git checkout %PUBLISH_BRANCH%
+
+IF ERRORLEVEL 1 (
+    :: Empty if not exist
+    rd /s /q .
+) ELSE (
+    :: Otherwise update it
+    rmdir /s /q .git
+)
+
+:: Reset errorlevel
+ver >nul
+
+IF NOT EXIST %PACKAGE% mkdir %PACKAGE%
+
+xcopy /S /E /Y ..\..\output\*.* %PACKAGE%\
+
+git config --global user.name "Azure DevOps"
+git config --global user.email peterghost86@gmail.com
```

Comment:
Is there an automated account that can be used rather than an account tied to an individual?

Reply:
No, I don't think I have one. But I can fake one if that's needed.